continuous_replication.go

```diff
@@ -216,6 +216,7 @@ func stateFnCatchingUp(r *ContinuousReplication) stateFnContinuous {
 				r.LastSequencePushed = stats.GetEndLastSeq()
 				r.ReplicationStats.AddDocsRead(stats.GetDocsRead())
 				r.ReplicationStats.AddDocsWritten(stats.GetDocsWritten())
+				r.ReplicationStats.AddNumAttachmentsTransferred(stats.GetNumAttachmentsTransferred())
```

Comment:
Do we need to add attachment bytes/docsChecked here as well?

Reply:
Yep.  Thanks for catching that